source/pages/pki/pkiRotate.rst

```diff
@@ -4,128 +4,144 @@
 
-Initial steps for all keys
-----------------------------------
+#. :ref:`Generate new signing and encryption private keys<pki_gen_private_key>` for both the VSP and MSA. This is 4 keys in total.
+#. :ref:`Generate a certificate signing request<pki_gen_csr>` for each of the the keys you generated in step 1.
+#. :ref:`Submit the certificate signing requests<pki_submit_csr>` from step 2 to the IDAP certificate authority.
 
-#. Generate a :ref:`new private key <pki_gen_private_key>`.
-#. Generate a :ref:`certificate signing request <pki_gen_csr>` and :ref:`submit it <pki_submit_csr>` to the IDAP certificate authority.
-#. The IDAP certificate authority issues the new certificate to the government service.
+The IDAP certificate authority will issue you a signed certificate for your MSA and VSP encryption and signing public keys.
 
 
-Rotate your service encryption key
------------------------------------
+.. _rotateMSA_encKey:
 
+Rotate your MSA encryption key
+-----------------------------------------------------
 
-#. Add the new private encryption key to your service endpoint. Your service can now use the new and old private keys to decrypt SAML messages.
-#. Send the new certificate to the GOV.UK Verify operations team.
-#. The GOV.UK Verify operations team installs the new certificate on the hub. The GOV.UK Verify hub now uses the new key to encrypt SAML messages for your service.
-#. Remove the old encryption key from your service endpoint.
+1. Install the new encryption certificate (.crt) and private encryption key (.pk8) on the MSA. To do this, add a second list item under ``encryptionKeys`` in your :ref:`MSA configuration <yamlfile>`. The list element you're adding contains the details for your new (2017) public and private key, for example:
 
+  .. code-block:: yaml
 
-Rotate your service signing key
-----------------------------------
+    encryptionKeys:
+    - publicKey:
+        certFile: msa_encryption_2016.crt
+        name: MSA Encryption 2016
+      privateKey:
+        keyFile: msa_encryption_2016.pk8
+    - publicKey:
+        certFile: msa_encryption_2017.crt
+        name: MSA Encryption 2017
+      privateKey:
+        keyFile: msa_encryption_2017.pk8
 
-#. Send the new signing certificate to the the GOV.UK Verify operations team.
-#. The GOV.UK Verify operations team installs the new certificate on the GOV.UK Verify hub. The GOV.UK Verify hub now trusts SAML messages signed with the new and old keys.
-#. Replace the old private signing key with the new key on your service endpoint. Your service now signs SAML messages with the new key only.
-#. Inform the GOV.UK Verify operations team that new key is live.
-#. The GOV.UK Verify operations team removes the old certificate from the GOV.UK Verify hub. The GOV.UK Verify hub now trusts SAML messages signed with the new key only.
+  The MSA can now use both the new (2017) and old (2016) keys to decrypt SAML messages.
```

Comment:
@andy-paine I know the VSP doesn't need the encryption cert, but should we recommend a restart here since it's a config change?

Reply:
As per chat, I'll mention restart after config change in the intro to avoid being repetitive.